crMeta/crMetadata.go

```diff
@@ -603,7 +613,21 @@ func getMetaFromMCResponse(lookupResp *base.SubdocLookupResponse) (cas uint64, c
 		}
 		if cvVer > cas {
 			// ver is initially the same as cas. Cas may increase later. So ver should never be greater than cas
-			err = fmt.Errorf("ver>cas, ver=%v,cas=%v,cvHex=%s,pv=%s,mv=%s", cvVer, cas, ver, pv, mv)
+			err = fmt.Errorf("ver>cas, cvCas=%v,ver=%v,cas=%v,cvHex=%s,pv=%s,mv=%s", cvCas, cvVer, cas, ver, pv, mv)
+			return
+		}
+	}
+	if len(cvCasHex) == 0 {
+		cvCas = 0
+	} else {
```

Comment:
Unnecessary as cvCas is always 0 unless you set it.
Simply do an `if len(cvCasHex) > 0`